public/index.html

```diff
@@ -1,11 +1,357 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <meta charset="utf-8">
-  <title>Photo Site</title>
-  <link rel="stylesheet" href="/app.css">
-</head>
-<body>
+  <head>
+    <meta charset="utf-8">
+    <title>Photo Site</title>
+    <link rel="stylesheet" href="/app.css">
+  </head>
+  <body>
+    <nav class="navbar">
+      <div class="navbar-left">
+        <h2 class="navbar-left__logo">
+          <span class="navbar__logo fa fa-arrow-left"></span>
+          <span class="navbar__logo">/</span>
+          <span class="navbar__logo fa fa-tint"></span>
+        </h2>
+        <div class="navbar-left__company">
+          <p class="navbar-left__name">Christiana Zana</p>
+          <p class="navbar-left__job-title">Photography</p>
+        </div>
+      </div>
+      <div class="navbar-right">
+        <div class="navbar-right__page">
+          <!-- <h3 class="navbar-right__page-options"></h3>
+          <h3 class="navbar-right__page-options">Work</h3>
+          <h3 class="navbar-right__page-options">Contact</h3> -->
 
-</body>
+          <a href="public/about.html" class="navbar-right__page-options">About</a>
+          <a href="public/work.html" class="navbar-right__page-options">Work</a>
+          <a href="public/contact.html" class="navbar-right__page-options">Contact</a>
+        </div>
+      </div>
+    </nav>
+    <header class="header-container">
+      <div class="header-image">
+        <div class="avatar-shading">
+          <img src="http://fillmurray.com/175/175" alt="" class="header-image__avatar">
+        </div>
+        <span class="header-image__fa-image fa fa-spinner fa-pulse"></span>
+        <h2 class="header-image__text">Murray</h2>
+      </div>
+    </header>
+    <div class="outer-container">
+      <div class="row">
+        <main class="content-photo">
+          <div class="content-photo__container">
+            <div class="content-photo__container__image">
+              <img class "content-photo__container__image-pic" src="http://fillmurray.com/400/500" alt="">
+            </div>
+            <ul class="content-photo__ul">
+              <li class="content-photo__ul-items fa fa-camera item1">
+                Canon Powershot S95
+              </li>
+              <li class="content-photo__ul-items fa fa-eye">
+                22.5 mm
+              </li>
+              <li class="content-photo__ul-items fa fa-chrome">
+                <em>f</em> / 5.6
+              </li>
+              <li class="content-photo__ul-items fa fa-power-off">
+                1/1000
+              </li>
+              <li class="content-photo__ul-items fa fa-cc-jcb">
+                80
+              </li>
+            </ul>
+          </div>
+          <div class="date-stamp">
```

Comment:
I would move the date and caption stuff into the `content-photo__container__image` div.